### DISCUSSION TITLE
将文档工具dumi 升级之2.x  版本& yarn 转到pnpm

### DISCUSSION THREAD

**Discussion by bobfw** [alibaba/formily#4007] [Ideas]
我想将formily 的dumi升级之2.x以适配新版本nodejs，和yarn 转为之pnpm，这是两件事，你是否同意这两项工作。

**Comment by bobfw** [comment]
@janryWang 我拿总站文档试了试，还不错
![image](https://github.com/alibaba/formily/assets/26560612/8bd30be9-5f24-4ea1-aaad-e22b4b9c4c71)

**Reply by yiyunwan** [reply]
可以提PR啊，感觉是好看很多，下面图标记得换下就好
